hw/mcu/stm/stm32f1xx/include/mcu/stm32f1_bsp.h

```diff
@@ -42,6 +42,12 @@ struct stm32f1_uart_cfg {
     IRQn_Type suc_irqn;                 /* NVIC IRQn */
 };
 
+/*
+ * Internal API for stm32f1xx mcu specific code.
+ */
+int hal_gpio_init_af(int pin, uint8_t af_type, enum hal_gpio_pull pull,
```

Comment:
This one is strange... I removed it intentionally because AFs work differently on stm32f1, and on `master` only `i2c` still uses it, so I can only assume you have `I2C_0: 1`. Btw, PR #953 already removed this dependency, but I'm OK with merging temporarily, and I'll remove it again with that PR.

Reply:
Hmm, that is strange.  I don't have `I2C_0` enabled.  My target is defined as follows:
```
targets/blinky-olimex-p103
    app=apps/blinky
    bsp=@apache-mynewt-core/hw/bsp/olimex-p103
    build_profile=debug
```